#header for synchronization
std_msgs/Header header

#phase space markers
visualization_msgs/MarkerArray LEDs
